hack/build-test-bin.sh

```diff
@@ -20,3 +20,6 @@ fi
 ginkgo build ./functests
 mkdir -p cnf-tests/bin
 mv ./functests/functests.test ./cnf-tests/bin/cnftests
+
+go build -o ./cnf-tests/bin/mirror cnf-tests/mirror/mirror.go
+git rev-list -1 HEAD > ./cnf-tests/bin/cnftests-sha.txt
```

Comment:
why we need this line?

Reply:
To bump a known version in the image.
It's useful when testing / asking for feedback once images go around, to make sure what version is running. I wasn't able to dump it in the test binary itself since ginkgo do not accept the ldflags command as the go buiild command does.